version

```diff
@@ -1 +1 @@
-dev
+v0.3
```

Comment:
please revert this change - this file is automatically updated when we trigger a release on GitHub